# awesome-sequencing-tech-papers

**A collection of papers on comparison of long and/or short read sequencing technologies (and data generated from the platforms).**
|Sequencing|Technologies|For|The | Win|
|:--:|:--:|:--:|:--:|:--:|
| [2022](https://github.com/Nazeeefa/awesome-sequencing-tech-papers#2022) |[2021](https://github.com/Nazeeefa/awesome-sequencing-tech-papers#2021) | [2020](https://github.com/Nazeeefa/awesome-sequencing-tech-papers#2020)| [2019](https://github.com/Nazeeefa/awesome-sequencing-tech-papers/blob/master/Sequencing_Tech_2019_Papers.md)| [2018](https://github.com/Nazeeefa/awesome-sequencing-tech-papers/blob/master/Sequencing_Tech_2018_Papers.md)|
  
**Key:**
  - Hi-C, chromosome conformation capture
  - ONT, Oxford Nanopore Technology
  - PacBio, Pacific Biosciences
    - SMRT, Single-Molecule Real-Time
    - CLR, Continuous Long Read
    - CCS, Circular Consensus Sequencing
    - Hi-Fi, High-Fidelity
  - ðŸ“š Review/Opinion Article
  - ðŸ§¬ Study involves human genome(s)
  - ðŸ¦  Prokaryotic genome(s)
  - ðŸŒ± Plant genome(s)
  - ðŸ’‰ Focused towards medical genetics
  - ðŸ›  Includes tool/software development
  
  
Contribute by submitting [pull requests](https://github.com/Nazeeefa/awesome-sequencing-tech-papers/pulls), or posting suggestions as [issues](https://github.com/Nazeeefa/awesome-sequencing-tech-papers/issues). Thank you.

---

## 2022

#

**Title:** ðŸ§¬ [Target enrichment long-read sequencing with adaptive sampling can determine the structure of the small supernumerary marker chromosomes](https://www.nature.com/articles/s10038-021-01004-x)

**Description:** Comparative analysis of two clinical samples

**Platforms:** HiSeq 1500 (2â€‰Ã—â€‰101â€‰bp) and, ONT's GridION using R9.4.1 flowcell (ligation sequencing kit SQK-LSK109)

#

**Title:** ðŸ§¬ An assessment of bioinformatics tools for the detection of human endogenous retroviral insertions in short-read genome sequencing data ()

#

## 2021

**Title:** ðŸ›  ðŸ§¬ [Haplotype-aware variant calling enables high accuracy in nanopore long-reads using deep neural networks](https://www.nature.com/articles/s41592-021-01299-w)

**Description:** Evaluation of variant calling performance and runtime comparison of haplotype-aware pipelines

**Platforms:** The study used the HG003 and HG004 samples (from [PrecisionFDA](https://doi.org/10.1101/2020.11.13.380741)); analysing the 35X coverage of Illumina NovaSeq reads, 35X PacBio HiFi, and 90X ONT.

#

**Title:** ðŸŒ± [Combined use of Oxford Nanopore and Illumina sequencing yields insights into soybean structural variation biology](https://www.biorxiv.org/content/10.1101/2021.08.26.457816v1)

**Description:** The study is about exploring structural variants (SVs) in soybean's transposable element (TE) . The authors developed a pipeline to find whether SVs at various sites were due to novel TE insertion and/or excision.

**Platforms:** ONT MinION (SQK-LSK109 genomic DNA ligation kit) with a FLO-MIN106D flowcell (R9 chemistry), and Illumina HiSeq 2500

#

**Title:** ðŸ¦  [Microbial metagenome-assembled genomes of the Fram Strait from short and long read sequencing platforms](https://peerj.com/articles/11721/)

**Platforms:** Comparison was performed on datasets from marine environmental samples usinng PacBio Sequel II and Illumina HiSeq 3000.

#

**Title:** ðŸ¦  [Benchmarking metagenomic classification tools for long-read sequencing data](https://www.biorxiv.org/content/10.1101/2020.11.25.397729v2)

**Platforms:** ONT and PacBio - See section 2.2 Test Datasets for information on data sources.

#

**Title:** ðŸ§¬ [Long-read cDNA sequencing identifies functional pseudogenes in the human transcriptome](https://www.biorxiv.org/content/10.1101/2021.03.29.437610v1)

**Platforms:** Comparison was performed between data from  Illumina Human BodyMap 2.0 Project and PacBio Sequel II

#

**Title:** ðŸ¦ ðŸ˜› [High molecular weight DNA extraction strategies for long-read sequencing of complex metagenomes](https://www.biorxiv.org/content/10.1101/2021.03.03.433801v2)

**Description:** The study involves samples from human tongue.

**Platforms:** Illumina MiSeq and NextSeq500 and ONT's MinION

#

**Title:** ðŸ§¬ [Targeted long-read sequencing identifies missing disease-causing variation](https://www.cell.com/ajhg/fulltext/S0002-9297(21)00230-5)

**Description:** Use of adaptive sampling via ONT and PacBio platforms

**Platforms:** ONT's GridION (with MinKNOW control software v18.04.1), PacBio CLR sequencing (SMRT Cell 1Ms on the Sequel platform using v3 chemistry), and PacBio HiFi sequencing (SMRT Cell 8M on a Sequel II instrument)

#

**Title:** ðŸ§¬ [Structural variant selection for high-altitude adaptation using single-molecule long-read sequencing](https://www.biorxiv.org/content/10.1101/2021.03.27.436702v1)

**Platforms:** ONT and PacBio

#

**Title:** ðŸ¦  [High-accuracy long-read amplicon sequences using unique molecular identifiers with Nanopore or PacBio sequencing](https://www.nature.com/articles/s41592-020-01041-y)

**Description:** Comparison of ONT R9.4.1, PacBio CCS, and ONT R10.3 UMI (Unique Molecular Identifier)

**Platforms:** ONT MinION and PacBio Sequel II

#

**Title:** ðŸ¦  [Benchmarking of long-read assemblers for prokaryote whole genome sequencing](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6966772/)

**Platforms:** ONT MinION (R9.4), PacBio RSII (CLR), and Illumina (for hybrid assemblies)


---

Contribute by submitting [pull requests](https://github.com/Nazeeefa/awesome-sequencing-tech-papers/pulls), or posting suggestions as [issues](https://github.com/Nazeeefa/awesome-sequencing-tech-papers/issues). Thank you.
