# awesome-sequencing-tech-papers

- **Long and/or Short Read Sequencing Technologies**
  - [2020](https://github.com/Nazeeefa/awesome-sequencing-tech-papers#2020)
  - [2019](https://github.com/Nazeeefa/awesome-sequencing-tech-papers#2019)
  - [2018](https://github.com/Nazeeefa/awesome-sequencing-tech-papers#2018)
  - [2017](https://github.com/Nazeeefa/awesome-sequencing-tech-papers#2017)
  - [2016](https://github.com/Nazeeefa/awesome-sequencing-tech-papers#2016)
- **[Single-Cell](https://github.com/Nazeeefa/awesome-sequencing-tech-papers#single-cell)**
  
Contribute by submitting [pull requests](https://github.com/Nazeeefa/awesome-sequencing-tech-papers/pulls), or posting suggestions as [issues](https://github.com/Nazeeefa/awesome-sequencing-tech-papers/issues).

### Long and/or Short Read Sequencing Technologies

#### 2020

1. [A technology-agnostic long-read analysis pipeline for transcriptome discovery and quantification](https://www.biorxiv.org/content/10.1101/672931v2)
2. [Comparative analysis of seven short-reads sequencing platforms using the Korean Reference Genome: MGI and Illumina sequencing benchmark for whole-genome sequencing](https://www.biorxiv.org/content/10.1101/2020.03.22.002840v1)
3. [A preliminary study on the potential of Nanopore MinION and Illumina MiSeq 16S rRNA gene sequencing to characterize building-dust microbiomes](https://www.nature.com/articles/s41598-020-59771-0)
4. [Long-read error correction: a survey and qualitative comparison](https://www.biorxiv.org/content/10.1101/2020.03.06.977975v2) - A comprehensive analysis of 29 tools using PacBio and nanopore data.
5. [Overcoming uncollapsed haplotypes in long-read assemblies of non-model organisms](https://www.biorxiv.org/content/10.1101/2020.03.16.993428v1) - Assessment of long-read assemblers on PacBio and nanopore reads from the non-model non-vertebrate organism Adineta *vaga*
6. [Long-read human genome sequencing and its applications](https://www.nature.com/articles/s41576-020-0236-x) - A review article that focuses on PacBio (SMRT) sequencing and Oxford Nanopore Technologies, and compare those two major long-read technologies with short-read sequencing platforms in terms of read accuracy, throughput, and cost.

#### 2019

1. [Comparative analysis of sequencing technologies for single-cell transcriptomics](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1676-5) 
2. [Comprehensive analysis of structural variants in breast cancer genomes using
single molecule sequencing](https://www.biorxiv.org/content/10.1101/847855v1)
3. [Chromosome-scale assembly comparison of the Korean Reference Genome KOREF from PromethION and PacBio with Hi-C mapping information](http://dx.doi.org/10.1093/gigascience/giz125)
4. [Long-Read Sequencing Emerging in Medical Genetics](https://www.frontiersin.org/articles/10.3389/fgene.2019.00426/full)
5. [Comparison of long-read sequencing technologies in the hybrid assembly of complex bacterial genomes](https://www.biorxiv.org/content/10.1101/530824v2)
6. [A critical comparison of technologies for a plant genome sequencing project](https://academic.oup.com/gigascience/article/8/3/giy163/5281243)
7. [Illumina and Nanopore methods for whole genome sequencing of hepatitis B virus (HBV)](https://www.nature.com/articles/s41598-019-43524-9)

#### 2018

1. [Single molecule real-time (SMRT) sequencing comes of age: applications and utilities for medical diagnostics](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5861413/)
1. [Single-Molecule Sequencing: Towards Clinical Applications](https://www.cell.com/trends/biotechnology/fulltext/S0167-7799(18)30204-X?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS016777991830204X%3Fshowall%3Dtrue)
2. [NGS Technologies and their Application to the Study and Control of Bacterial Infection](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5857210/)
3. [Targeted Long-Read Sequencing of a Locus Under Long-Term Balancing Selection in *Capsella*](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5873921/)
4. [Real-time analysis of nanopore-based metagenomic sequencing from infected orthopaedic devices](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-5094-y) (Comparison between Illumina MiSeq and MinION)
5. [High-throughput targeted long-read single cell sequencing reveals the clonal and transcriptional landscape of lymphocytes](https://www.biorxiv.org/content/10.1101/424945v1.full) (2018)

#### 2017
1. [A Review on the Applications of Next Generation Sequencing Technologies as Applied to Food-Related Microbiome Studies](https://www.frontiersin.org/articles/10.3389/fmicb.2017.01829/full)
1. [Hybrid sequencing and map finding (HySeMaFi): optional strategies for extensively deciphering gene splicing and expression in organisms without reference genome](https://www.nature.com/articles/srep43793)
1. [DNA Sequencing Technologies: 2006-2016](https://www.nature.com/articles/nprot.2016.182)

#### 2016
1. [Comparison of NGS technologies for the comprehensive assessment of full-length hepatitis C viral genomes](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5035407/)
1. [The sequence of sequencers: The history of sequencing DNA](https://www.sciencedirect.com/science/article/pii/S0888754315300410)

Feel free to contribute by submitting [pull requests](https://github.com/Nazeeefa/awesome-sequencing-tech-papers/pulls), or posting suggestions as [issues](https://github.com/Nazeeefa/awesome-sequencing-tech-papers/issues).
